docs/framework/wcf/feature-details/hosting-in-internet-information-services.md

```diff
@@ -11,10 +11,6 @@ One option for hosting Windows Communication Foundation (WCF) services is inside
 ## Versions of IIS  
  WCF can be hosted on the following versions of IIS on the following operating systems:  
```

Comment:
This change is fine but I probably wouldn't mix the token replacement and the article updates.

This would have to be updated further to be complete. e.g. https://support.microsoft.com/en-us/help/224609/how-to-obtain-versions-of-internet-information-server-iis